api/src/main/java/com/epam/pipeline/manager/datastorage/providers/aws/s3/S3Helper.java

```diff
@@ -300,6 +303,44 @@ public void restoreFileVersion(String bucket, String path, String version) {
         moveS3Object(client, bucket, new MoveObjectRequest(path, version, path));
     }
 
+    public void restoreFolderVersion(String bucket, String path, RestoreFolderVO restoreFolderVO) {
+        AmazonS3 client = getDefaultS3Client();
+        try (S3ObjectDeleter deleter = new S3ObjectDeleter(client, bucket)) {
+            getDeletedItems(bucket, path, true, null, null, restoreFolderVO)
```

Comment:
Do we really need to path  `true, null, null` to this method?

Reply:
fixed